tests/fixtures/fixtures.py

```diff
@@ -8,9 +8,13 @@
 
 class GarageTestCase(unittest.TestCase):
```

Comment:
I think we should make clear that this is a TfTestCase. There is more than one backend in garage.

Reply:
sgtm